data/json/monsters/jabberwock.json

```diff
@@ -19,6 +19,7 @@
     "melee_dice": 2,
     "melee_dice_sides": 4,
     "melee_damage": [ { "damage_type": "cut", "amount": 0 } ],
+    "families": [ "prof_intro_biology", "prof_wp_demihuman" ],
```

Comment:
This is described as "an amalgamation". Maybe prof_wp_amalgamation?

Reply:
No. Amalgamations as referred to with weakpoints are a very specific *thing*, like a zombie, as opposed to being a general category, like a humanoid. Specifically, they're what happens when creatures too small to zombify have their corpses shaped by the blob into weird and alien forms. All these monsters loooong predate the addition of amalgamations.